### ISSUE TITLE
Ginger tabs are not displayed for standard and virt users

### ISSUE THREAD

**Issue by sureshab** [kimchi-project/ginger#332]
Installed wok, gingerbase, kimchi, ginger, gingers390x.
logged in with user having standard user profile (with login shell and without login shell). In Wok tab, Settings is displayed with "User Activity Log"
In Virtualization - Guests, Storage, Network tabs are displayed.
But in Host - Dashboard tab is shown but its blank. and none of the other ginger tabs are displayed. 
Is it the expected behavior?
Same results when logged in with virtuser.
![screenshot from 2016-05-04 12 02 33](https://cloud.githubusercontent.com/assets/12499230/15006728/b80c52f4-11f0-11e6-9a0e-e5f328a74f6c.png)

![screenshot from 2016-05-04 12 07 13](https://cloud.githubusercontent.com/assets/12499230/15006738/d14fa5ae-11f0-11e6-8c78-cd8483a08aff.png)

![screenshot from 2016-05-04 12 07 20](https://cloud.githubusercontent.com/assets/12499230/15006751/e1d6a134-11f0-11e6-8722-d4205017cf06.png)

**Comment by danielhb** [comment]
Hi @sureshab , sorry for the 'small' delay into seeing this :p 

I don't believe we tested this kind of scenario properly. Both 'virt-users' and 'standard' should have access to the Dashboard contents (not sure about generating debug reports). This is something we need to look at.

As far as Ginger tab goes, a virt/standard user shouldn't be messing with host configuration in my opinion, but it's up to debate. We could show the features in 'read-only' mode instead of hiding them.

**Comment by sureshab** [comment]
@danielhb, yes dashboard contents like statistics, host info should be shown. but other tabs needs discussion. I agree with read-only mode at minimum.
